hadoop-hdds/common/src/main/java/org/apache/hadoop/utils/db/TypedTable.java

```diff
@@ -205,6 +205,16 @@ public String getName() throws IOException {
     return rawTable.getName();
   }
 
+  @Override
+  public long getEstimatedKeyCount() throws IOException {
+    if (rawTable instanceof RDBTable) {
+      return rawTable.getEstimatedKeyCount();
+    }
+    throw new IllegalArgumentException(
+        "Unsupported operation getEstimatedKeyCount() on table type " +
+            rawTable.getClass().getCanonicalName());
```

Comment:
Does IllegalArgumentException look good here?

Reply:
I don't think we need a check here, as the underlying table for TypedTable is rocksdbTable.

Reply:
Thanks for that. I've pushed a commit for this.